### ISSUE TITLE
missing once() or exactlyOnce()

### ISSUE THREAD

**Issue by CaseyB** [mockito/mockito#3014]
Mockito provides nice convenience methods for checking that a method in a mock is called at least once, or at most once, but there is nothing for ensuring that it is called exactly once.

For comparison:
```
// Ensure that start is called at least once
verify(mockServer, atLeast(1)).start()
// becomes
verify(mockServer, atLeastOnce()).start()

// Ensure that start is not called more than once
verify(mockServer, atMost(1)).start()
// becomes
verify(mockServer, atMostOnce()).start()

// Ensure that start is called at only once
verify(mockServer, times(1)).start()
// should be
verify(mockServer, exactlyOnce()).start()
```

The name could be `once()`, `onlyOnce()`, or `exactlyOnce()`. Any of these would improve the readability of the code.

**Comment by TimvdLippe** [comment]
Duplicate of #2166
